pyspecProcScripts/third_level/rough_table_of_integrals.py

```diff
@@ -80,7 +81,6 @@ def rough_table_of_integrals(
         ax=ax1,
     )
     ax1.set_title("Signal pathway / ph0")
-    ax1.set_ylabel(r"$\beta$ / $\mathrm{\mu s \sqrt{W}}$")
```

Comment:
this ruins the generality of this function (which should work for all sorts of stuff, and should be (is) handled by pyspecdata)

Reply:
was trying to get the y axis labels to use the latex symbol but figured out how to do symbols with latex symbols so like you said no need for this!